frontend/src/css/cat-theme.css

```diff
@@ -145,3 +145,20 @@ Progess Steps
 .nav-pills .show > .nav-link {
   background-color: #00565e;
 }
+
+/* Side Navigation */
+
+.side-nav-grid {
+  display: -ms-grid;
```

Comment:
Just curious: why are we putting these in the css and not in a style const?

Reply:
As discussed offline, I did not find a way to put this type of display in a const. I tried putting like _display: "-ms-grid"_ but it did not work. It seems like we need to put this specific type of display in a _.css_ file to be recognized.

@caleybrock, do you have an idea on how to put that in the component itself?

Reply:
Can you make another CSS file called `side-nav.css` with this new CSS and then import that file into SideNavigation.jsx?

Reply:
Fixed in commit #https://github.com/code-for-canada/project-thundercat/pull/103/commits/eea0ce44bd90b0a1fa3b852972de8ed35f120dc1